WordPress/wordpress-playground
# WordPress Build: Use Playground CLI to produce minified WordPress build (without the SQLite integration plugin)

Uses Playground CLI to produce the minified WordPress builds:

```bash
$ bun \
    packages/playground/cli/src/cli.ts run-blueprint \
    --wp=https://wordpress.org/6.5.zip \
    --mount-before-install=./wordpress-site:/wordpress
```

Those new builds ships **without** the SQLite database integration plugin but **with** the SQLite database file containing all the data and tables created by a vanilla WordPress installation.

Advantages:

* Less code to maintain – Playground CLI replaces a custom bash script relying on wget, unzip, sed, wp-cli etc.
* More testing for our internal tools – we're dogfooding Playground CLI and the boot protocol in the critical code path. 

## Testing instructions

* Confirm the CI checks passed – this is about as good of a testing as we can get.
* Run `npm run dev` and confirm the local Playground loads without any issues.

Run `bun packages/playground/cli/src/cli.ts server --login` and confirm it loads an installed WordPress.

Related: #1398
